modules/misunderstood/src/views/full/MainScreen.tsx

```diff
@@ -0,0 +1,300 @@
+import { Button, ButtonGroup, HTMLTable, Intent } from '@blueprintjs/core'
+import clsx from 'clsx'
+import React from 'react'
+
+import {
+  ApiFlaggedEvent,
+  ContextMessage,
+  DbFlaggedEvent,
+  FLAGGED_MESSAGE_STATUS,
+  RESOLUTION_TYPE,
+  ResolutionData
+} from '../../types'
+
+import style from './style.scss'
+import { RESOLUTION } from './util'
+
+const DeletedList = ({
+  events,
+  totalEventsCount,
+  undeleteEvent
+}: {
+  events: DbFlaggedEvent[]
+  totalEventsCount: number
+  undeleteEvent: (id: string) => void
+}) => (
```

Comment:
When you have multiple elements in your typing, I recommend moving them to an interface, it's easier to read the code afterwards (it keeps them on one-line)